atom/browser/browser_win.cc

```diff
@@ -54,14 +56,20 @@ bool GetProtocolLaunchPath(mate::Arguments* args, base::string16* exe) {
     *exe = path.value();
   }
 
-  // Read in optional args arg
-  std::vector<base::string16> launch_args;
-  if (args->GetNext(&launch_args) && !launch_args.empty())
-    *exe = base::StringPrintf(L"\"%s\" %s \"%%1\"",
+  if (launch_args.empty()) {
+    base::string16 formatString = includeOriginalArg ?
+      L"\"%s\" \"%%1\"" :
+      L"\"%s\"";
```

Comment:
@kevinsawicki I think it's still consistent. It was quoted before:
``` c++
*exe = base::StringPrintf(L"\"%s\" \"%%1\"", exe->c_str());
```

The command line args were never quoted though:
```
L"\"%s\" %s \"%%1\""
L"\"%s\" %s"
```